### PR TITLE
Add Diarization documentation

### PR DESCRIPTION
**Pull request by kizitomayanja**
This PR adds diarization documentation.

---

### Testing
To test, follow these steps

```sh
git checkout Diarization_documentation
mkdocs serve
```
Then browse to `localhost:8000`

image

### CODE REVIEW

File: docs/tutorials/13-diarization.md [hunk 1]
```diff
@@ -0,0 +1,292 @@
...
+
+```
+
+**Running the diarization model**
+```python
+diarize_model = DiarizationPipeline(use_auth_token=hf_token, device=device)
+diarize_segments = diarize_model("/content/kitakas_eng.mp3", min_speakers=1, max_speakers=2)
+
+diarize_segments
+```
```
Comment (ak3ra) [inline docs/tutorials/13-diarization.md]: could we show sample outputs of what diarization looks like after this cell?

**Review by ak3ra — COMMENTED** [review]
Looks good to me! Just left one comment for your consideration.

File: docs/tutorials/13-diarization.md (hunk 1 above)
Comment (kizitomayanja) [reply docs/tutorials/13-diarization.md]: Thank you for the feedback.

We have added a few sample outputs at the end of the documentation, one for after `diarize segments` and another for after `assign_word_speakers`

Comment (ak3ra) [reply docs/tutorials/13-diarization.md]: Looks great now, will proceed to Merge this PR 
🚀

**Pull request merged**